src/parser/jobs/mnk/modules/AoE.tsx

```diff
@@ -11,14 +11,12 @@ export class AoE extends AoEUsages {
 	// You awake to find yourself enlightened to the true power of AoE
 	suggestionIcon = this.data.actions.ENLIGHTENMENT.icon
 
-	// Assuming user is in the correct Form
-	// Technically, FPF is 3 however if the buff is about to fall off, it's 2.
-	// Tracking remaining duration is kind of a pain here due to Anatman so eh.
+	// SotD is on-par with DK at 2 targets, but needs 3 to beat out Boot
 	trackedActions: AoEAction[] = [
 		{
-			aoeAction: this.data.actions.ARM_OF_THE_DESTROYER,
+			aoeAction: this.data.actions.SHADOW_OF_THE_DESTROYER,
 			stActions: [this.data.actions.BOOTSHINE, this.data.actions.DRAGON_KICK],
-			minTargets: 3,
+			minTargets: 2,
```

Comment:
This should always be 3+ since DK + boot wins over 2 GCDs. E.g. with current gear (20.7% crit rate, 1.56 crit modifier):

SotD ePotency = `220*1.56 + (340 / 5)*(1 + (.56 * .207))` = 419.1p
DK ePotency = `320*(1 + (.56 * .207)) + (.207)*(340 / 5)*(1 + (.56 * .207))` = 372.8p
Boot ePotency = `310*1.56 + (340 / 5)*(1 + (.56 * .207))` = 559.5p

2x SotD = 838.2p
DK + boot = 932.3p

Reply:
Huh weird, I set it to 3 but looks like a stash pop borked it, good catch.